src/System.Private.Uri/tests/FunctionalTests/UriParserTest.cs

```diff
@@ -0,0 +1,556 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
+// Author:
```

Comment:
If you have made significant changes to the tests, you need to add this line:

// Copyright (c) .NET Foundation and Contributors